poi-ooxml/src/main/java/org/apache/poi/openxml4j/opc/OPCPackage.java

```diff
@@ -1463,24 +1474,36 @@ public boolean validatePackage(OPCPackage pkg) throws InvalidFormatException {
      * @see #save(OutputStream)
      */
     public void save(File targetFile) throws IOException {
```

Comment:
Idk if GH is glitching, but I feel this line should be deleted too? Replaced with the `public void save(Path targetPath)` below.

Reply:
We don't have the luxury of being the only consumers. Others may still desire this API